pages/index.vue

```diff
@@ -35,6 +39,12 @@
       hero() {
         return this.page.primaryImageOfPage ? this.page.primaryImageOfPage.fields : null;
       },
+      heroDescription() {
+        return this.hero.description || this.page.description;
+      },
+      heroHeadline() {
+        return this.page.headline || this.hero.headline || this.page.name;
+      },
```

Comment:
I guess we are going to remove the hero banner at some point, but should hero and page go in the same order in both heroDescription and heroHeadline?

Reply:
Good point. Done in 0b73a23